db/schema.rb

```diff
@@ -10,20 +10,20 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_11_24_105520) do
+ActiveRecord::Schema.define(version: 2023_01_03_132957) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "activities", force: :cascade do |t|
-    t.integer "trackable_id"
+  create_table "activities", id: :serial, force: :cascade do |t|
     t.string "trackable_type"
-    t.integer "owner_id"
+    t.integer "trackable_id"
```

Comment:
it seems the default ordering of polymorphic associations has changed. Probably it will be expensive to alter all of these columns, so I will try to change the default back to the old one first.